dao/subscriptions.js

```diff
@@ -11,6 +11,13 @@ function notFoundError(id) {
   );
 }
 
+function endpointNotFoundError(endpoint) {
```

Comment:
On a middle-term perspective, I wonder if we shouldn't factorize this type of function. Something like:
```js
function notFoundError(subject, discriminant) {
  return new NotFoundError(
    `${subject}_not_found`,
    `The ${subject} with ${discriminant.name} \`${discriminant.value}\` does not exist.`
}
```

What do you think?